mobile/mobile.go

```diff
@@ -40,32 +38,47 @@ type Messenger interface {
 	Notify(event *Event)
 }
 
-// Wrapper is the object exposed in the frameworks
-type Wrapper struct {
-	RepoPath  string
-	messenger Messenger
-}
-
 // NodeConfig is used to configure the mobile node
+// NOTE: logLevel is one of: CRITICAL ERROR WARNING NOTICE INFO DEBUG
 type NodeConfig struct {
 	RepoPath      string
 	CentralApiURL string
 	LogLevel      string
 	LogFiles      bool
 }
 
-// NewNode is the mobile entry point for creating a node
-// NOTE: logLevel is one of: CRITICAL ERROR WARNING NOTICE INFO DEBUG
-func NewNode(config *NodeConfig, messenger Messenger) (*Wrapper, error) {
-	var m Mobile
-	return m.NewNode(config, messenger)
+// Mobile is the name of the framework (must match package name)
+type Mobile struct {
+	RepoPath  string
+	Mnemonic  string
+	Online    <-chan struct{} // not readable from bridges
```

Comment:
used by tests that need to run when online... the bridge / rn layer get's notified via the `messenger`